src/utils/errorHandler.js

```diff
@@ -0,0 +1,23 @@
+const logger = require('./logger');
+
+// error handler middleware that catches all errors
+const errorHandler = (err, req, res, next) => {
+  let { statusCode } = err;
+  const { message, stack } = err;
+
+  // displays the error and the location it occurred
+  logger.error(message);
+  logger.info(stack);
+
+  // if no status code has been set, set to 500 as default
+  if (!statusCode) statusCode = 500;
```

Comment:
@kelue I'm somehow not comfortable with this code but I will update it and you see what I did same with the data I told you to set to default

Reply:
Alright, I'll make sure to study the update so I can understand what I needed to improve on better for next time